tools/find_dali.cmake

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##################################################################
+# Utility function, useful when building custom C++ library where
+# DALI is used as a shared library. Assumes, that DALI wheel is
+# installed within the system. Acquires paths to DALI libraries
+# and dependencies introduced by the wheel. On the event, that
+# multiple DALI wheels are installed, picks the most recent one.
```

Comment:
It picks the last installed, not the most recent one.

Reply:
Fixed